sunpy/timeseries/sources/rhessi.py

```diff
@@ -123,15 +123,15 @@ def _parse_hdus(cls, hdulist):
         header = MetaDict(OrderedDict(header))
         data = DataFrame(d['data'], columns=d['labels'], index=d['time'])
         # Add the units data
-        units = OrderedDict([('3 - 6 keV', u.dimensionless_unscaled),
-                             ('6 - 12 keV', u.dimensionless_unscaled),
-                             ('12 - 25 keV', u.dimensionless_unscaled),
-                             ('25 - 50 keV', u.dimensionless_unscaled),
-                             ('50 - 100 keV', u.dimensionless_unscaled),
-                             ('100 - 300 keV', u.dimensionless_unscaled),
-                             ('300 - 800 keV', u.dimensionless_unscaled),
-                             ('800 - 7000 keV', u.dimensionless_unscaled),
-                             ('7000 - 20000 keV', u.dimensionless_unscaled)])
+        units = OrderedDict([('3 - 6 keV', u.ct / u.s / u.Unit('detector')),
```

Comment:
what is this detector unit? how is this going to fall out if people want to use this data for something else?